src/viam/sdk/common/version_metadata.cpp

```diff
@@ -0,0 +1,41 @@
+#include <viam/sdk/common/version_metadata.hpp>
+
+#include <sstream>
+
+#include <viam/sdk/common/private/version_metadata.hpp>
+
+namespace viam {
+namespace sdk {
+
+std::string sdk_version() {
+    std::string version_metadata(impl::k_version);
+    version_metadata.erase(0, version_metadata.find(';') + 1);
+    return version_metadata.substr(0, version_metadata.find(';'));
```

Comment:
Maybe have the function body as
```cpp
static std::string result = []{
    std::string version_metadata(impl::k_version);
    version_metadata.erase(0, version_metadata.find(';') + 1);
    return version_metadata.substr(0, version_metadata.find(';'));
}();

return result;
```

That way we only ever do this computation once. 

This might seem like a micro optimization, but if this stuff later gets plugged into, eg, the response type of a grpc request that throws out the version, it could easily be hit over and over

Reply:
ooh good thinking! thanks :)